trigger/process.go

```diff
@@ -314,12 +313,12 @@ func TriggerDownstreamProjectsForPush(ctx context.Context, projectId string, eve
 				DownstreamProject:            ref,
 				ConfigFile:                   trigger.ConfigFile,
 				TriggerType:                  model.ProjectTriggerLevelPush,
+				TriggerID:                    trigger.Project,
 				DefinitionID:                 trigger.DefinitionID,
 				Alias:                        trigger.Alias,
-				ProjectID:                    trigger.Project,
 				UnscheduleDownstreamVersions: trigger.UnscheduleDownstreamVersions,
 				PushRevision: model.Revision{
-					Revision:        utility.FromStringPtr(event.GetHeadCommit().SHA),
+					Revision:        utility.FromStringPtr(event.GetHeadCommit().ID),
```

Comment:
Why this change?

Reply:
It is the correct field to reference, `SHA` is not guaranteed to be defined: [docs](https://docs.github.com/en/webhooks/webhook-events-and-payloads#push)